pkg/controller/logstorage/logstorage_controller_test.go

```diff
@@ -314,6 +317,31 @@ var _ = Describe("LogStorage controller", func() {
 
 						mockStatus.AssertExpectations(GinkgoT())
 					})
+
+					It("should update the default settings for DNS logs", func() {
```

Comment:
I don't think this test adds much value, maybe we can simply remove it. There are several places in this repo where we test that the client can get and update resources.

Reply:
Done